Watchman.Discord/Areas/Protection/Controllers/AdministrationController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.Protection.BotCommands;
+using Watchman.Discord.Areas.Protection.Services;
+
+namespace Watchman.Discord.Areas.Protection.Controllers
+{
+    public class AdministrationController : IController
+    {
+        private readonly MessagesServiceFactory _messagesServiceFactory;
+        private readonly ComplaintsChannelService _complaintsChannelService;
+
+        public AdministrationController(MessagesServiceFactory messagesServiceFactory, ComplaintsChannelService complaintsChannelService)
+        {
+            this._messagesServiceFactory = messagesServiceFactory;
+            this._complaintsChannelService = complaintsChannelService;
+        }
+
+        [AdminCommand]
+        public async Task CreateChannelForComplaints(ComplaintsChannelCommand command, Contexts contexts)
+        {
+            var complaintsChannelName = string.IsNullOrWhiteSpace(command.Name) ? "skargi" : command.Name;
+            _ = await this._complaintsChannelService.CreateComplaintsChannel(complaintsChannelName, contexts);
```

Comment:
o co chodzi z tym _ ?

Reply:
daje sie to jak nie chcesz mieć zmiennej bo jej nie potrzebujesz 
ale coś musisz podać żeby był jakiś punkt odniesienia

jak dobrze pamiętam w c# 7 to wprowadzili

Reply:
używam tego aby wskazać, że dana zmienna (zwrócona przez metodę) nie jest w tym miejscu potrzebna, aby ktoś później nie musiał się zastanawiać czy to było zamierzone

Reply:
aha czyli to jest tylko po to aby ktoś nie zadawał pytania w stylu "czy to nie powinno być zapisane do jakieś zmiennej? czy zwracana wartość tej metody nie jest nam potrzebna?".

Reply:
tak